R/orcutt-tidiers.R

```diff
@@ -29,11 +29,11 @@
 #' @family orcutt tidiers
 #' @seealso [orcutt::cochrane.orcutt()]
 tidy.orcutt <- function(x, ...) {
-  warning("deal with tidy.orcutt conf.int nonsense")
-  dots <- enquos(...)
-  dots$conf.int <- FALSE
-
-  rlang::exec(tidy.lm, x, !!!dots)
+  s <- summary(x)
+  co <- stats::coef(s)
+  nn <- c("estimate", "std.error", "statistic", "p.value")
+  ret <- fix_data_frame(co, nn[1:ncol(co)])
```

Comment:
Much cleaner, I love it! I'm also trying to move away from using `fix_data_frame()`. I think if you use `ret <- as_tibble(..., rownames = "term")` and then set colnames on `ret` we could be able to get rid of the `fix_data_frame()` call.